src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -53,6 +53,56 @@ public async Task UnidirectionalAndBidirectionalChangeValues()
             Assert.Equal(20, serverConnection.GetRemoteAvailableUnidirectionalStreamCount());
         }
 
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/52048")]
```

Comment:
These 2 new tests pass on their own but if run within the whole suite they cause the crash with stream being finalized after the connection. That should be fixed with https://github.com/dotnet/runtime/pull/52800, I'll revisit it enabling them afterwards.

Reply:
We discussed offline that it is caused by not accepted stream getting stuck inside accept queue and not being disposed there. You can try accepting it (after all test checks if it's important for the test) and disposing manually, or you may wait for Tomas' fix, your choice.